frontend/src/lib/components/CommandPalette/commandPaletteLogic.tsx

```diff
@@ -567,6 +567,28 @@ export const commandPaletteLogic = kea<commandPaletteLogicType>({
                         : [],
             }
 
+            const debugCopySessionRecordingURL: Command = {
+                key: 'debug-copy-session-recording-url',
+                scope: GLOBAL_COMMAND_SCOPE,
+                resolver: {
+                    icon: IconRecording,
+                    display: 'Debug: Copy the session recording link to clipboard',
+                    executor: () => {
+                        const LOOK_BACK = 30
+                        const recordingStartTime = Math.max(
```

Comment:
Do you need to pass the start time explicitly? Looks like the share recording link doesn't have it so I would just remove it.

Reply:
Ok that's a lookback, right!

Reply:
We don't have to have it, but if you've had PostHog open for 2 hours it's going to be hard to find the relevant bit. This starts you 30 seconds before when you share it (let me know if you think I can make the code more obvious for this)